src/templates/template_exports/index-page-template.js

```diff
@@ -115,13 +110,9 @@ export const IndexPageTemplate = ({
           <h1>Latest stories</h1>
           <BlogRoll blogRoll={blogRoll} />
         </section>
-        <section className="testimonials component">
+        <section className="family-reviews component">
           <h2>What Families Are Saying About The Coding Space</h2>
-          <div className="testimonials__reviews">
-            {reviews.map((review, index) => (
-              <Review {...review} key={index} />
```

Comment:
@aasalazar23 `.review` was a "leakier" class than I believe you anticipated

In particular, the min/max sizes overrode the natural grid size 

For flex and grid-based layouts, it's best practice to have the width-defining style live "up" in the layout rather than on the item directly